File: fizzbuzz/python/Arnold/fizzbuzz.py
import sys


def fizz_buzz(number):
    return_string = ""
    if number < 1:
        return "Invalid number. Please input number greater than 0."

    for i in range(1, number + 1):
        string_build = ""
        if i % 3 == 0:
            string_build += "Fizz"
        if i % 5 == 0:
            string_build += "Buzz"
        if i % 7 == 0:
            string_build += "Ping"
        if i % 11 == 0:
            string_build += "Pong"
        if len(string_build) == 0:
            string_build += str(i)
        return_string += string_build + "\n"
    return return_string


if __name__ == "__main__":
    num = int(sys.argv[1])
    print(fizz_buzz(num))

# Test cases: 0, 3, 5, 15, 7, 21, 35, 105, 11, 33, 55, 165, 77, 231, 385, 1155

File: fizzbuzz/C#/Anthony/fizzbuzz.cs
using System;

namespace Anthony
{
    class fizzBuzz
    {
        static void Main(string[] args)
        {
            int number;
            bool test = int.TryParse(args[0], out number);
            var fizz = new fizzBuzz();
            fizz.FizzBuzz(number);
        }

        public String FizzBuzz(int testNumber){
            String numberString ="";
            for(int i=1; i<testNumber+1; i++){
                numberString ="";
                if (i%3==0){
                    numberString = numberString+"Fizz";
                }
                if(i%5==0){
                    numberString = numberString+"Buzz";
                }
                if(i%7==0){
                    numberString = numberString+"Ping";
                }
                if(i%11==0){
                    numberString = numberString+"Pong";
                }
                if(numberString.Length==0){
                    numberString = numberString+i.ToString();
                }
                Console.WriteLine(numberString);
            }
            return numberString;
        }
    }
}
